src/components/BlipCard.vue

```diff
@@ -530,9 +532,15 @@ export default {
     resolveUnsupportedRepliedType() {
       if (this.document.type === 'application/vnd.lime.reply+json') {
         const { replied } = this.document.content
-        const isUnsupportedRepliedType = !supportedRepliedTypes.includes(replied.type)
+        let isSupportedRepliedType = supportedRepliedTypes.includes(replied.type)
 
-        if (isUnsupportedRepliedType) {
+        if (isSupportedRepliedType &&
+            replied.type === MessageTypesConstants.MEDIALINK_MESSAGE) {
+          isSupportedRepliedType = supportedRepliedMediaTypes
+            .findIndex(type => replied.value.type.indexOf(type) !== -1) !== -1
+        }
```

Comment:
```suggestion
       if (replied.type === MessageTypesConstants.MEDIALINK_MESSAGE) {
          const repliedMediaType = replied.value.type
          isSupportedRepliedType = supportedRepliedMediaTypes.some(type => repliedMediaType.includes(type))
        }
```

I think you can use `some` and `includes` function instead of `findIndex` to simplify and make it more readable because you don't need to check the index. What do you think?